stdlib/3/asyncio/futures.pyi

```diff
@@ -41,16 +41,17 @@ class Future(Awaitable[_T], Iterable[_T]):
     if sys.version_info >= (3, 7):
         def get_loop(self) -> AbstractEventLoop: ...
         def _callbacks(self: _S) -> List[Tuple[Callable[[_S], Any], Context]]: ...
+        def add_done_callback(self: _S, fn: Callable[[_S], Any], context: Optional[Context] = ...) -> None: ...
```

Comment:
The new argument is keyword-only:
```
In [208]: asyncio.Future.add_done_callback??                                                                                                         
Signature: asyncio.Future.add_done_callback(self, fn, /, *, context=None)
```
Also, it looks like `fn` is positional-only, so while you're at it you could change its stub to `__fn` (the notation for positional-only args in stubs).